### DISCUSSION TITLE
About employee

### DISCUSSION THREAD

**Discussion by Kushaldotel** [horilla-opensource/horilla#100] [General]
Like when I create a new horilla user: manage.py createhorilla user like I get loggedin. 
when I create employee like there is no password section. Is there no feature like employee can login ClockIn and clock out?

**Comment by horilla-opensource** [comment]
Hi @Kushaldotel ,
Be default on initial creation of the employee, the username is their email id and password will be their phone number, which can be reset by the employee after first login.
This is the case with the employee data creation directly from the employee view.
In the case of new users from the recruitment or onboarding, they will be provided with a portal through which they will be able to register their password.

**Reply by Kushaldotel** [reply]
Thanks that's great. I will explore more and try to contribute too.
